sdk/eventhub/azure-eventhub/azure/eventhub/_consumer_client.py

```diff
@@ -229,23 +244,28 @@ def receive(self, on_event, **kwargs):
          bool as the value indicating whether the starting_position for a specific partition is inclusive or not.
          This can also be a single bool value for all starting_position. The default value is False.
         :paramtype starting_position_inclusive: bool or dict[str,bool]
-        :keyword on_error: The callback function which would be called when there an error occurs during receiving.
+        :keyword on_error: The callback function which would be called when error is raised during receiving
+         after retry attempts exhaust and during the process of load balance.
```

Comment:
The callback function that will be called when an error is raised during receiving 
after retry attempts are exhausted, or during the process of load-balancing.

Reply:
thanks, updated